### ISSUE TITLE
Roadmap / ideas for the future

### ISSUE THREAD

**Issue by mmbotelho** [okTurtles/group-income#991]
### Problem

On this issue, you will find a list of ideas for the future development of Group Income. This is not, in any way, a binding list of tasks to be completed. The purpose of the issue is to document ideas that have been discussed internally but were never written down.

### Solution

The priority ATM should be to get the prototype ready to launch. Below, you will find a list of features that are **NOT included in the prototype**. These may be already designed and ready to implement, or they can be just ideas that were not fully developed yet.

These are currently being discussed and edited here: https://gitlab.okturtles.org/okturtles/group-income-simple/-/wikis/Roadmap/ideas-for-the-future

In the future we will publish a public roadmap once we have it more solidified.

**Comment by CChavinha** [comment]
Hey! The link (https://gitlab.okturtles.org/okturtles/group-income-simple/-/wikis/Roadmap/ideas-for-the-future) isn't working. 
![404-Page-Not-Found](https://user-images.githubusercontent.com/66070659/93917474-f3301d00-fd02-11ea-8626-ba4fc91aff56.png)

**Comment by taoeffect** [comment]
Yeah you need to be signed into Gitlab. I just added you to that project, give it another try!
